doc/build-unix.md

```diff
@@ -79,13 +58,11 @@ install necessary parts of boost:
 
         sudo apt-get install libboost-all-dev
 
-BerkeleyDB is required for the wallet.
-
-**For Ubuntu only:** db4.8 packages are available [here](https://launchpad.net/~bitcoin/+archive/bitcoin).
+BerkeleyDB is required for the wallet. db4.8 packages are available [here](https://launchpad.net/~bitcoin/+archive/bitcoin).
```

Comment:
As commented in https://github.com/ElementsProject/elements/pull/134#issuecomment-284870475 this shouldn't be true for elements unless I'm missing something. But even with --with-incompatible-bdb it seems we require them, so I guess we can change the comment to "we're working on removing the bdb dependency for the wallet too" or something.